crates/wg_network/src/node.rs

```diff
@@ -5,11 +5,6 @@ use std::rc::Rc;
 
 pub type NodeRef = Rc<RefCell<Node>>;
 
-#[derive(Debug, Clone)]
-pub struct Topology {
-    pub nodes: Vec<NodeRef>,
-}
-
```

Comment:
There is another PR that is removing this part. Please stick only to one argument per PR.
Same with the other file you deleted.

So please revert these changes.